src/common/transformations/tests/CMakeLists.txt

```diff
@@ -29,18 +29,27 @@ ov_add_test_target(
     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
     DEPENDENCIES
     LINK_LIBRARIES
-        gtest
-        gtest_main
-        openvino::runtime::dev
-        commonTestUtils
-        lptNgraphFunctions
         gmock
         funcTestUtils
-        inference_engine_legacy
         offline_transformations
         sharedTestClasses
+        lptNgraphFunctions
     INCLUDES
         $<TARGET_PROPERTY:inference_engine_obj,SOURCE_DIR>/src
     LABELS
         TRANSFORMATIONS
-)
\ No newline at end of file
+)
+
+file(GLOB_RECURSE legacy_tests
+    "${CMAKE_CURRENT_SOURCE_DIR}/legacy/*.cpp" # CVS-55385
+    )
+
+set_source_files_properties(${legacy_tests} PROPERTIES INCLUDE_DIRECTORIES
+    $<TARGET_PROPERTY:inference_engine_legacy,INTERFACE_INCLUDE_DIRECTORIES>)
+
+if(WIN32)
+    string(REPLACE "/Zi" "/Z7" CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG}")
+    string(REPLACE "/Zi" "/Z7" CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG}")
+    string(REPLACE "/Zi" "/Z7" CMAKE_C_FLAGS_RELWITHDEBINFO "${CMAKE_C_FLAGS_RELWITHDEBINFO}")
```

Comment:
Should we keep this replace for Release and ReleaseWithDebug info. It looks Release should be intact. Please check if modification of release with debug infor really required

Reply:
I just made back changes for this: https://github.com/openvinotoolkit/openvino/blob/bc685ac8a0be3c35eb1d887f075c2aac061b084c/src/common/transformations/CMakeLists.txt#L62-L67
We have only Debug and Release configurations on CI so I didn't test ReleaseWithDebugInfo. It requires a lot of time to check because the issue has repro rate 5-15%. But I can launch it this night to be sure we need or not need of these changes.

Reply:
OK, just make changes for Debug only

Reply:
I tested RelWithDebugInfo config yesterday. It failed due to error in vpux_mlir_compiler_static.lib.
So I suppose we can remove any change for this config until it is fixed.